CRM/Upgrade/Incremental/sql/5.52.2.mysql.tpl

```diff
@@ -0,0 +1,3 @@
+{* file to handle db changes in 5.52.2 during upgrade *}
+
+ALTER TABLE `civicrm_dashboard_contact` MODIFY COLUMN `is_active` TINYINT(4) DEFAULT 0 COMMENT 'Is this widget active?';
```

Comment:
So if someone upgrades 5.52.0 => 5.52.1 => [5.53.0](https://github.com/civicrm/civicrm-core/pull/24238), this statements runs twice... but that seems safe to me... ie it should be a nullop on the second run.

Reply:
Confirmed while `r-run`ing.

I'll do a separate PR for updating release notes.

Merge on pass.